react-frontend/src/App.js

```diff
@@ -1,48 +1,48 @@
-import React, { Component } from 'react';
-import Swimlane from './Swimlane.js'
+import React, {Component} from "react";
+import Swimlane from "./Swimlane.js";
+import TaskData from "./model/TaskData";
+import SwimlaneData from "./model/SwimlaneData";
 
 export default class App extends Component {
 
     constructor() {
         super();
+        let tempSwimlaneData = new Map();
+        tempSwimlaneData.set('Gauss', new SwimlaneData('Gauss', [
+            new TaskData('Revert "ETD"', 'http://wp.pl', 12312123123),
+            new TaskData('t2', 'http://wp.pl', 12312123123)
+        ]));
+        tempSwimlaneData.set('Maxwell', new SwimlaneData('Maxwell', [
+            new TaskData('Add stuff', 'http://wp.pl', 12312123123),
+        ]));
+        tempSwimlaneData.set('Planck', new SwimlaneData('Planck', [
+            new TaskData('Gradle magic', 'http://wp.pl', 12312123123),
+            new TaskData('t2', 'http://wp.pl', 12312123123),
+            new TaskData('t3', 'http://wp.pl', 12312123123),
+            new TaskData('t4', 'http://wp.pl', 12312123123),
+        ]));
+
         this.state = {
-            teams: [
-                {
-                    name: 'Gauss',
-                    tasks: [
-                        { title: 'Revert "ETD"', url: 'http://wp.pl', startTime: '12312123123' },
-                        { title: 't2', url: 'http://wp.pl', startTime: '12312123123' },
-                    ]
-                },
-                {
-                    name: 'Maxwell',
-                    tasks: [
-                        { title: 'Add stuff', url: 'http://wp.pl', startTime: '12312123123' },
-                    ]
-                },
-                {
-                    name: 'Planck',
-                    tasks: [
-                        { title: 'Gradle magic', url: 'http://wp.pl', startTime: '12312123123' },
-                        { title: 't2', url: 'http://wp.pl', startTime: '12312123123' },
-                        { title: 't3', url: 'http://wp.pl', startTime: '12312123123' },
-                        { title: 't4', url: 'http://wp.pl', startTime: '12312123123' },
-                    ]
-                },
-            ]
-        }
+            allSwimlaneData: tempSwimlaneData
+        };
     }
 
     render() {
         return (
             <div className="App">
                 <h1 className="title">RotateIT</h1>
-                {this.state.teams.map(this.makeSwimlane)}
+                {this.printAllSwimlaneData(this.state.allSwimlaneData)}
             </div>
         );
     }
 
-    makeSwimlane(team) {
-        return (<Swimlane title={team.name} tasks={team.tasks} />);
+    printAllSwimlaneData(allSwimlaneData) {
+        let result = [];
+        allSwimlaneData.forEach((swimlaneData) => result.push(this.makeSwimlane(swimlaneData)));
```

Comment:
Or in other words, `map(...)` is only a function available in Array's (so that some array may be 'mapped' to some other array)